src/components/templates/TemplateFooter.vue

```diff
@@ -346,6 +353,10 @@ export default {
     color: #888d93;
```

Comment:
```
.privacy,
.tnc {
    color: #888d93;
}
```

use the above shorthand if multiple css have same styling

or you can create only a single class (like footer-link) and use in both the places